pyvista/jupyter/pv_pythreejs.py

```diff
@@ -537,6 +537,20 @@ def convert_plotter(pl):
                     width += pv_ren.width + pv_ren.border_width * 2
                 grid[i, j] = convert_renderer(pv_ren)
 
+        # check for linked cameras
+        for i in range(n_row):
+            for j in range(n_col):
+                pv_ren = pl.renderers[j + n_row * i]
+                camera = pv_ren.camera
+
+                # check if the camera is cloned with any other cameras
+                for i_view in range(i, n_row):
+                    for j_view in range(j + 1, n_col):
+                        if camera is pl.renderers[j_view + n_row * i_view].camera:
+                            # must link both camera and controls
+                            grid[i_view, j_view].camera = grid[i, j].camera
+                            grid[i_view, j_view].controls = grid[i, j].controls
```

Comment:
I'm not a fan of expressions that use many `for` statements. However, I am convinced that this expression is currently the best for this process. We can think of a way to split this part into functions to avoid reducing readability. What do you think about it?

Reply:
Fixed in f53da9bdb1e26ad2a795134f38f6e7189121dcc2.